string NAME=active_region
bool ERASE_WHEN_FALSE=1

ObjectLocationID location_id_sw
ObjectLocationID location_id_ne

bool truth_value
